src/renderer/components/state-table.tsx

```diff
@@ -216,6 +288,22 @@ export class StateTable extends React.Component<
     );
   }
 
+  private renderRootState(): JSX.Element | null {
+    const content = (
+      <JSONView
+        data={this.state.rootStateData}
+        raw={this.state.rawRootState}
```

Comment:
issue(blocking): If we're already passing the parsed data into the JSON view via `props.data`, `props.raw` will be unused according to:

https://github.com/tinyspeck/sleuth/blob/4a290fd25882b2667f8da8c0659e19eeee7ae875/src/renderer/components/json-view.tsx#L22

We shouldn't need to store the raw data in a separate state value.

Reply:
This makes sense, so would we also not store need to store the raw data in the overall state as they are used in the same way https://github.com/tinyspeck/sleuth/blob/4a290fd25882b2667f8da8c0659e19eeee7ae875/src/renderer/components/state-table.tsx#L90C1-L90C1

Reply:
I think the `raw` value is useful there because `data` and `raw` are both nullable values. I think you either pass one or the other. I _think_ the use-case for `raw`  is to pass in strings rather than JSON.